lib/command/workers/runTests.js

```diff
@@ -1,23 +1,20 @@
 const tty = require('tty');
 
-if (!tty.getWindowSize) {
+if ('getWindowSize' in tty) {
```

Comment:
`!tty.getWindowSize` this syntax usually using.

Reply:
should i back this to initial condition, assertion and initialization of "getWindowSize" ?

Reply:
will back